parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -241,20 +244,63 @@ public static Statistics toParquetStatistics(org.apache.parquet.column.statistic
       if(statistics.hasNonNullValue()) {
         stats.setMax(statistics.getMaxBytes());
         stats.setMin(statistics.getMinBytes());
+        setBloomFilter(stats, statistics);
      }
     }
     return stats;
   }
 
-  public static org.apache.parquet.column.statistics.Statistics fromParquetStatistics(Statistics statistics, PrimitiveTypeName type) {
-    // create stats object based on the column type
-    org.apache.parquet.column.statistics.Statistics stats = org.apache.parquet.column.statistics.Statistics.getStatsBasedOnType(type);
+  private static void setBloomFilter(Statistics stats,
+      org.apache.parquet.column.statistics.Statistics statistics) {
+    if (!(statistics instanceof BloomFilterStatistics)) {
+      return;
+    }
```

Comment:
I don't follow why we need this test and define the signature of this method this way.
can't we just convert from one type to the other?